.circleci/config.yml

```diff
@@ -50,12 +53,13 @@ jobs:
       - attach_workspace:
           at: .
       - run:
-          name: Lint and format code and sort imports
-          # ruff check --select I . : check linting and imports sorting without fixing (to fix, use --fix)
-          # ruff format --check . : check code formatting without fixing (to fix, remove --check)
+          name: Lint code, format code and sort imports
           command: |
-            poetry run ruff check --select I .
-            poetry run ruff format --check .
+            source .venv/bin/activate
+            # Check linting and imports sorting (to fix, add --fix)
+            ruff check
+            # Check code formatting (to fix, add --check)
```

Comment:
Nit: his comment looks weird, it says to add `--check`, but the line below does already have the `--check`

Reply:
Changed!